.travis.yml

```diff
@@ -67,6 +67,8 @@ env:
     CFG_API_CUSTOMER_NUMBER="11223344"
     CFG_API_SHOP_NUMBER="12345"
     CFG_API_KEY="11111111111111111111"
+    CFG_MINIMUM_ORDER_AMOUNT="21"
```

Comment:
you can remove this config from travis & magento_setup.sh, just hardcode in .magerun file.